src/mscorlib/src/System/Reflection/Emit/GenericTypeParameterBuilder.cs

```diff
@@ -192,7 +192,7 @@ public override Type MakeArrayType(int rank)
 
         public override Type GetGenericTypeDefinition() { throw new InvalidOperationException(); }
 
-        public override Type MakeGenericType(params Type[] typeArguments) { throw new InvalidOperationException(SR.Arg_NotGenericTypeDefinition); }
+        public override Type MakeGenericType(params Type[] typeArguments) { throw new InvalidOperationException(SR.Format(SR.Arg_NotGenericTypeDefinition, this.GetType().FullName)); }
```

Comment:
not sure if this.GetType().FullName makes sense

Reply:
@AtsushiKan  ?